cmd/tg_bot/internal/environment.go

```diff
@@ -0,0 +1,48 @@
+package internal
+
+import (
+	"log"
+
+	"gopkg.in/telebot.v3"
+	"nodemon/pkg/entities"
+	"nodemon/pkg/storing/chats"
+)
+
+type TelegramBotEnvironment struct {
+	ChatStorage *chats.Storage
+	Bot         *telebot.Bot
+	Mute        bool
+}
+
+func NewTelegramBotEnvironment(bot *telebot.Bot, storage *chats.Storage, shutUp bool) *TelegramBotEnvironment {
+	return &TelegramBotEnvironment{Bot: bot, ChatStorage: storage, Mute: shutUp}
+}
+
+func (tgEnv *TelegramBotEnvironment) Start() {
+	log.Println("Telegram bot started")
+	tgEnv.Bot.Start()
+	log.Println("Telegram bot finished")
+}
+
+func (tgEnv *TelegramBotEnvironment) SendMessage(msg []byte) {
+	if tgEnv.Mute {
+		return
+	}
+
+	chatID, err := tgEnv.ChatStorage.FindChatID(entities.TelegramPlatform)
+	if err != nil && err != chats.ErrorChatNotFound {
+		log.Printf("failed to find chat id: %v", err)
+		return
+	}
+
+	if err == chats.ErrorChatNotFound {
+		log.Println("have not received a chat id yet")
+		return
+	}
```

Comment:
I think it looks prettier.
```suggestion
	if err != nil {
		if errors.Is(err, chats.ErrorChatNotFound) {
			log.Println("have not received a chat id yet")
		} else {
			log.Printf("failed to find chat id: %v", err)
		}
		return
	}
```